src/main/java/teammates/ui/controller/InstructorFeedbacksPageAction.java

```diff
@@ -17,22 +17,23 @@
     
     @Override
     protected ActionResult execute() throws EntityDoesNotExistException {
-        //This can be null. Non-null value indicates the page is being loaded 
-        //   to add a feedback to the specified course
+        // This can be null. Non-null value indicates the page is being loaded 
+        // to add a feedback to the specified course
         String courseIdForNewSession = getRequestParamValue(Const.ParamsNames.COURSE_ID);
         String feedbackSessionNameForSessionList = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
         String isUsingAjax = getRequestParamValue(Const.ParamsNames.IS_USING_AJAX);
         
         new GateKeeper().verifyInstructorPrivileges(account);
                 
-        if (courseIdForNewSession!=null) {
+        if (courseIdForNewSession != null) {
             new GateKeeper().verifyAccessible(
                     logic.getInstructorForGoogleId(courseIdForNewSession, account.googleId), 
-                    logic.getCourse(courseIdForNewSession), Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
+                    logic.getCourse(courseIdForNewSession),
+                    Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
         }
 
         InstructorFeedbacksPageData data = new InstructorFeedbacksPageData(account);
-        data.isUsingAjax = (isUsingAjax == null) ? false : true;
+        data.isUsingAjax = isUsingAjax != null;
```

Comment:
I'm fine with reviews for these files since I don't intend on changing them until the review phase.
Do you mean `data.isUsingAjax = (isUsingAjax != null)`?
I suppose it might be clearer that way since it now resembles `data.isUsingAjax = isUsingAjax = null` slightly.